src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/EndreEndringstidspunkt.tsx

```diff
@@ -47,7 +47,7 @@ const EndreEndringstidspunkt: React.FC<{
                     settVisModal(true);
                 }}
             >
-                <Calender />
+                <CalendarIcon fontSize={'1.4rem'} />
```

Comment:
Klarer ikke å få frem denne fordi åpenBehandling.endringstidspunkt har blitt satt til null i alle tilfellene jeg har prøvd. @halvorbmundal kjenner du til om åpenBehandling sin endringstidspunkt-verdi fortsatt skal bli satt med deres nye kode?

Reply:
Hm. Vet at jeg har fått opp denne knappen senest denne uka. Jeg jobber i alle fall med logikken rundt dette as we speak, så det burde i alle fall funke når jeg har dunket ut de neste PRene